Sources/Valet/SecureEnclave.swift

```diff
@@ -21,7 +21,6 @@
 import Foundation
 
 
-@available(macOS 10.11, *)
```

Comment:
Our minimum supported target on macOS is macOS 10.11. So there is no need to specify availability in this case.